rs/bitcoin/ckbtc/minter/Cargo.toml

```diff
@@ -40,6 +40,7 @@ lazy_static = { workspace = true }
 minicbor = { workspace = true }
 minicbor-derive = { workspace = true }
 num-traits = { workspace = true }
+regex = "1.11.0"
```

Comment:
Since this is a dependency only used for testing purposes, it should go under `[dev-dependencies]`.